Eatery/AppDelegate.swift

```diff
@@ -1,9 +1,10 @@
-import UIKit
-import SwiftyJSON
-import Fabric
 import Crashlytics
+import Fabric
```

Comment:
If we have completed the setup for FireBase Analytics, is it still necessary to import Fabric or is the plan to keep it until Google deprecates?